VLogProcessor/libs/mac_client/opt/vertica/include/verticaodbc.h

```diff
@@ -1,32 +0,0 @@
-/*
```

Comment:
@ShahabT I cannot make promise but I guess somebody just downloaded the whole build archive of vertica and copy one of the critical jar file to dependency directory. An evidence is that the whole project is written in java using jdbc to connect to the db server but there is bunch of header files like "verticaodbc.h " remaining. thx:)